### ISSUE TITLE
Import settings from terminal

### ISSUE THREAD

**Issue by okian** [sadko4u/lsp-plugins#48]
Add ability to import settings from terminal like:
```bash
 $ lsp-plugins-para-equalizer-x16-mono --import ~/lsp-settings/eq.cfg
```

**Comment by sadko4u** [comment]
Hi there!
This option is already implemented in *devel* branch and will be accessible in the 1.1.10 release.

It will look like this:
```
lsp-plugins-para-equalizer-x16-mono --help
Usage: lsp-plugins-para-equalizer-x16-mono [parameters]

Available parameters:
  -c, --config <file>   Load settings file on startup
  -h, --help            Output help
```

**Comment by sadko4u** [comment]
Available in 1.1.10 release!
